File: app/src/main/java/com/challenge/fidoreader/Util/ArrayCustomList.java
package com.challenge.fidoreader.Util;

import java.util.ArrayList;

public class ArrayCustomList<Object> extends ArrayList {

    int expectedCount = 0;

    public void setExpectedCount(int cnt){
        this.expectedCount = cnt;

    }

    public int getExpectedCount(){
        return expectedCount;
    }

}

File: app/src/main/java/com/challenge/fidoreader/Util/ArrayCustomList.kt
package com.challenge.fidoreader.Util

import java.util.*

class ArrayCustomList<Object> : ArrayList<Any?>() {
    var expectedCountNum: Int = 0

    fun setExpectedCount(cnt: Int) {
        expectedCountNum = cnt
    }

    fun getExpectedCoun(): Int{
        return expectedCountNum
    }
}